src/tools/crawler/main.rs

```diff
@@ -53,6 +53,10 @@ struct Args {
     /// If present, start an RPC server at the specified address
     #[clap(short, long, value_parser)]
     rpc_addr: Option<SocketAddr>,
+
+    /// Default port used for connecting to the nodes
+    #[clap(short, long, value_parser, default_value_t = ZCASH_P2P_DEFAULT_MAINNET_PORT)]
+    default_port: u16,
```

Comment:
maybe the logic can be: 

default ports are 8233 and 18233 by definition, which we assign here to a listening port
```suggestion
    /// Listening port used for connecting to the nodes for which the port is unknown.
    /// If not provided, a default port for the mainnet/testnet network will be used.
    #[clap(short, long, value_parser, default_value_t = ZCASH_P2P_DEFAULT_MAINNET_PORT)]
    listening_port: u16,
```

Reply:
Maybe `node_listening_port` to not confuse anyone that it's our listening port?

Reply:
sounds even better!

Reply:
Done